src/editors/data/redux/app/selectors.test.js

```diff
@@ -0,0 +1,100 @@
+// import * in order to mock in-file references
+import { blockTypes } from '../../constants/app';
+import * as urls from '../../services/cms/urls';
+import * as selectors from './selectors';
+
+jest.mock('reselect', () => ({
+  createSelector: jest.fn((preSelectors, cb) => ({ preSelectors, cb })),
+}));
+jest.mock('../../services/cms/urls', () => ({
+  unit: (args) => ({ unit: args }),
+}));
+
+const testState = { some: 'arbitraryValue' };
+const testValue = 'my VALUE';
+const testKey = 'MY key';
+
+describe('app selectors unit tests', () => {
+  const {
+    appSelector,
+    simpleSelectors,
+  } = selectors;
+  describe('appSelector', () => {
+    it('returns the app data', () => {
+      expect(appSelector({ ...testState, app: testValue })).toEqual(testValue);
+    });
+  });
+  describe('simpleSelectors', () => {
+    const testSimpleSelector = (key) => {
+      test(`${key} simpleSelector returns its value from the app store`, () => {
+        const { preSelectors, cb } = simpleSelectors[key];
+        expect(preSelectors).toEqual([appSelector]);
+        expect(cb({ ...testState, [key]: testValue })).toEqual(testValue);
+      });
+    };
+    describe('simple selectors link their values from app store', () => {
+      [
+        'blockContent',
+        'blockId',
+        'blockType',
+        'blockValue',
+        'courseId',
+        'editorInitialized',
+        'saveResponse',
+        'studioEndpointUrl',
+        'unitUrl',
+      ].map(testSimpleSelector);
+    });
+  });
+  describe('returnUrl', () => {
+    it('is memoized based on unitUrl and studioEndpointUrl', () => {
+      expect(selectors.returnUrl.preSelectors).toEqual([
+        simpleSelectors.unitUrl,
+        simpleSelectors.studioEndpointUrl,
+      ]);
+    });
+    it('returns urls.unit with the unitUrl if loaded, else an empty string', () => {
+      const { cb } = selectors.returnUrl;
```

Comment:
Just curious, what does "cb" refer to as an acronym

Reply:
callback